Project.toml

```
@@ -2,7 +2,7 @@ name = "JLSO"
 uuid = "9da8a3cd-07a3-59c0-a743-3fdc52c30d11"
 license = "MIT"
 authors = ["Invenia Technical Computing Corperation"]
-version = "2.2.0"
+version = "2.2.1"
```

Changes should be rebased an this should be update to 2.3.1:
```suggestion
version = "2.3.1"
```